master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+{: .alert .alert-danger }
+
+1. Add a new IP pool.  
+   **Note**: The new IP pool must be within the same cluster CIDR.
+1. Disable the old IP pool.  
+   **Note**: Disabling an IP pool only prevents new IP address allocations; it does not affect the networking of existing pods.
+1. Delete pods from the old IP pool.  
+   This includes any new pods that may have been created with the old IP pool prior to disabling the pool.
+1. Verify that new pods get an address from the new IP pool.
+1. Delete the old IP pool.
+
+### Tutorial
+
+In the following example, we created a Kubernetes cluster using **kubeadm**. When we installed calico nodes in this cluster, a default IP pool was assigned: **192.168.0.0/16**. Now, we want pods in the cluster to use IPs in the CIDR: **10.0.0.0/16** (within the cluster CIDR). 
+
+Let’s run `calicoctl get ippool -o wide` to see the default IP pool, **default-ipv4-ippool**.
+
+<pre>
+NAME                  CIDR             NAT    IPIPMODE   VXLANMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     Never       false
+</pre>
+
+When we run `calicoctl get wep --all-namespaces`, we see that a pod is created using the default range (192.168.52.130/32).
+
+<pre>
+NAMESPACE     WORKLOAD                   NODE      NETWORKS            INTERFACE
+kube-system   coredns-6f4fd4bdf-8q7zp   vagrant   192.168.52.130/32   cali800a63073ed
+</pre>
+
+Let’s get started changing this pod to the new IP pool (10.0.0.0/16).
+
+#### Step 1: Add a new IP pool
+
+We add a new **IPPool** with the CIDR range, **10.0.0.0/16**.
+
+<pre>
+cat <<EOF | calicoctl apply -f -
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+name: new-pool
+spec:
+cidr: 10.0.0.0/16
+ipipMode: Always
+natOutgoing: true
+EOF
+</pre>
+
+Let’s verify the new IP pool.
+
+`calicoctl get ippool -o wide`
+
+<pre>
+NAME                  CIDR             NAT    IPIPMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     false
+new-pool              10.0.0.0/16      true   Always     false
+</pre>
+
+#### Step 2: Disable the old IP pool
+
+List the existing IP pool definition.
+
+`calicoctl get ippool -o yaml > pool.yaml`
+
+<pre>
+apiVersion: projectcalico.org/v3
+items:
+- apiVersion: projectcalico.org/v3
+  kind: IPPool
+  metadata:
+    name: default-ipv4-ippool
+  spec:
+    cidr: 192.0.0.0/16
+    ipipMode: Always
+    natOutgoing: true
+- apiVersion: projectcalico.org/v3
+  kind: IPPool
+  metadata:
+    name: new-pool
+  spec:
+    cidr: 10.0.0.0/16
+    ipipMode: Always
+    natOutgoing: true
+</pre>
+
+Edit pool.yaml.
+
+Disable this IP pool by setting: `disabled: true`
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+  name: default-ipv4-ippool
+spec:
+  cidr: 192.0.0.0/16
+  ipipMode: Always
+  natOutgoing: true
+  disabled: true
+</pre>
+
+Apply the changes. 
+
+Remember, disabling a pool only affects new IP allocations; networking for existing pods is not affected.
+
+`calicoctl apply -f pool.yaml`
```

Comment:
I think these should all be in triple backticks for the proper formatting.

Reply:
Fixed.